include/gsl/span

```diff
@@ -408,15 +400,19 @@ public:
     }
 
     template <std::size_t N, class ArrayElementType = std::remove_const_t<element_type>>
+    // GSL_SUPPRESS(bounds.4) // NO-FORMAT: attribute // TODO: parser bug
     constexpr span(std::array<ArrayElementType, N>& arr) noexcept
         : storage_(&arr[0], details::extent_type<N>())
     {
+
```

Comment:
In other places you are replacing "{\n}" by "{}", here you are adding a new line?

Reply:
thanks for catching!